ChannelMonitor/channel-monitor.py

```diff
@@ -161,7 +161,7 @@ def _build_log(channel: str, user: str, action: str, timestamp: str) -> Dict:
             "user": user,
             "action": action,
             "timestamp": timestamp,
-            "string_repr": f"{timestamp}: {channel} was {action}'d by {user}",
```

Comment:
You need to update the test that corresponds with this message
```

=================================== FAILURES ===================================
________________________________ test_build_log ________________________________

testbot = <errbot.backends.test.TestBot object at 0x7f0bcdbe2790>

    def test_build_log(testbot):
        plugin = testbot.bot.plugin_manager.get_plugin_obj_by_name("ChannelMonitor")
        log = plugin._build_log(CHANNEL, USER, "create", 12345)
    
        assert log["channel"] == CHANNEL
        assert log["user"] == USER
        assert log["action"] == "create"
        assert log["timestamp"] == 12345
>       assert log["string_repr"] == f"12345: {CHANNEL} was create'd by {USER}"
E       assert '12345: @test...reated #test.' == "12345: #test...'d by @tester"
E         - 12345: #test was create'd by @tester
E         + 12345: @tester created #test.
```

Reply:
Good catch. Updated.